admin/osx/pre_install.sh.cmake

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
+#exec   > ~/owncloud-install-pre.log
+# exec  2>&1
+# BASH_XTRACEFD=1
+# set -x
 
-OC_INSTANCE=$(ps aux | grep "/Applications/@APPLICATION_EXECUTABLE@.app/Contents/MacOS/@APPLICATION_EXECUTABLE@")
+# don't grep in one line, to avaoid grepping the grep process...
+PROCESSES=$(ps aux)
+OC_INSTANCE=$(echo "${PROCESSES}" | grep "/Applications/@APPLICATION_EXECUTABLE@.app/Contents/MacOS/@APPLICATION_EXECUTABLE@")
```

Comment:
i like this - my first attempt probably would have been to use `grep -v grep` breaking brandings containing grep in the name. ^_°